app/services/event_mailer.rb

```diff
@@ -161,6 +164,12 @@ def handle_message(delivery_info, _metadata, payload)
     unlisten if db_connection_error?(e)
   end
 
+  def fetch_user(user)
+    return User.find_by(uid: user.uid) if user.uid.present?
```

Comment:
В связи с тем, что дальше не проверяешь, что пришёл не null, то, имхо, так лучше.
```suggestion
    return User.find_by!(uid: user.uid) if user.uid.present?
```